src/layout/MainLayout.tsx

```diff
@@ -0,0 +1,11 @@
+
+function MainLayout() {
+
+  return (
+    <>
+     <h1>Hello World!</h1>
+    </>
+  )
+}
+
+export default MainLayout;
```

Comment:
layouts usualy accept a child to render with in their tags
like <MainLayout>
<some component> // children
</MainLayout>

Reply:
changed done
![scr2](https://github.com/user-attachments/assets/bf7c1b0b-36ac-4f35-94af-021f2e946541)

Reply:
Changed done 
![scr2](https://github.com/user-attachments/assets/4f9fb8d9-decd-496c-b20d-bd343980fa13)